Gemfile

```diff
@@ -27,17 +27,18 @@ gem 'modernizr-rails'
 
 # Use Unicorn as the app server
 # gem 'unicorn'
+gem 'puma'
```

Comment:
It was thought it could make the musical scores demo site a little more responsive. Also, Puma replaces webrick in Rails 5 because of ActionCable, so this is a bit of future proofing.